config/initializers/action_cable_patch.rb

```diff
@@ -0,0 +1,45 @@
+ActiveSupport.on_load(:action_cable) do
+  require 'action_cable/subscription_adapter/postgresql'
+  ActionCable::SubscriptionAdapter::PostgreSQL.class_eval do
+    def new_connection
```

Comment:
That sounds reasonable @matthewd.  @skateman Can you put a warn in dev mode if rails version is >= 5.0.1 to check if this patch is still needed?